bungee/src/main/java/me/leoko/advancedban/bungee/BungeeMethods.java

```diff
@@ -277,6 +280,12 @@ public String getInternUUID(Object player) {
 
     @Override
     public String getInternUUID(String player) {
+        if (luckPermsSupport) {
+            UUID uuid = LuckPermsProvider.get().getUserManager().lookupUniqueId(player).join();
```

Comment:
Are you sure this is guaranteed to not throw NoClassDefFoundError when LuckPerms is not present? I have found that sometimes classloading still occurs despite using a branch.

Reply:
Yes, this will probably cause errors for users that don't use LuckPerms. There should be no import statements of the LuckPerms API in java files that are also in use without LuckPerms. 

We have avoided this issue for permissions checks with the `LuckPermsOfflineUser` class.